src/ctelnet.cpp

```diff
@@ -3013,7 +3013,7 @@ void cTelnet::setKeepAlive(int socketHandle)
     setsockopt(socketHandle, IPPROTO_TCP, TCP_KEEPIDLE, &timeout, sizeof(timeout));
 #endif
 
-    // Interval between keep-alives, in seconds:
+    // Interval between keep-alive, in seconds:
```

Comment:
This is not necessarily a typo.

Reply:
Agreed. Each of them is a "keep-alive" and you'd need two of them have a period *between* them...
:grinning: